modules/nf-core/gunc/mergecheckm/tests/nextflow.config

```diff
@@ -0,0 +1,5 @@
+process {
+  withName: CHECKM_QA {
+    ext.args = "--tab_table"
+  }
+}
```

Comment:
There is nicer method of specifying these now with instructions merged in last week, so they are next to the test definition, try it out!

https://nf-co.re/docs/guidelines/components/modules#configuration-of-extargs-in-tests

Reply:
I  tried it but setting the params inside `run()` doesn't seem to work (and it's not covered in the docs), so I left the args in `nextflow.config` again.